storage/client_raft_test.go

```diff
@@ -524,9 +523,11 @@ func TestReplicateAfterTruncation(t *testing.T) {
 		if err != nil {
 			return false
 		}
-		getResp := reply.(*roachpb.GetResponse)
-		log.Infof("read value %d", mustGetInt(getResp.Value))
-		return mustGetInt(getResp.Value) == 39
+		val := mustGetInt(reply.(*roachpb.GetResponse).Value)
+		if log.V(1) {
```

Comment:
I'd just remove this log line instead of making it a `V(1)`. I don't think `V(1)` logs make much sense in tests because by default logs are only printed from failing tests and if a test is failing when would you not want the extra log detail?